internal/cephfs/volume.go

```diff
@@ -36,6 +38,10 @@ var (
 
 const (
 	cephEntityClientPrefix = "client."
+
+	// modeAllRWX can be used for setting permissions to Read-Write-eXecute
```

Comment:
Read-Write-eXecute to Read-Write-Execute

Reply:
RWX - the X comes from e**X**ecute :smile:

Reply:
:facepalm: